src/utils/dayjs.ts

```diff
@@ -44,3 +44,14 @@ export const formatDayJs = (time: dayjs.Dayjs, mode: 'locale' | 'utc' = 'locale'
 
   return time.local().format('MMMM DD, YYYY hh:mm:ss A (z)');
 };
+
+export function msToTime(ms) {
+  const seconds = (ms / 1000).toFixed(0);
+  const minutes = (ms / (1000 * 60)).toFixed(0);
+  const hours = (ms / (1000 * 60 * 60)).toFixed(0);
+  const days = (ms / (1000 * 60 * 60 * 24)).toFixed(0);
+  if (seconds < 60) return `${seconds} Sec`;
+  if (minutes < 60) return `${minutes} Min`;
+  if (hours < 24) return `${hours} Hrs`;
+  return `${days} Days`;
+}
```

Comment:
This shouldn't be in dayjs. We currently have helpers to display `nanoToSeconds` and `secondsToDays` in `src/utils/time` please refer to `src/screens/params` to look at how they are used with **dayjs**. The function created above is not i18n friendly

Reply:
Ok, didn't see those functions. I removed msToTime method and used secondsToDays in lock tokens message type